po/es.po

```diff
@@ -404,19 +404,19 @@ msgstr ""
 
 #: ../src/login/org.freedesktop.login1.policy.in.h:47
 msgid "Hibernate the system while an application asked to inhibit it"
-msgstr "Hibernar el sistema a pesar de que una aplicación lo impide"
+msgstr "Hibernar el sistema a pesar de que una aplicación lo impide."
```

Comment:
Please don't end translated sentences with periods if the original sentence doesn't.